components/automate-ui/src/app/modules/infra-proxy/infra-tab-change/infra-tab/infra-tab.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-infra-tab',
+  template: `
+  <div [hidden]="!active" class="test">
+    <ng-content></ng-content>
+  </div>
+`
+})
+// TODO:eng-ex The active tab should be tracked in the parent and not the individual tab
+export class InfraTabComponent {
+  @Input() tabTitle: string;
+  @Input() active = false;
+  @Input() disabled = true;
+}
```

Comment:
Instead of making an almost exact duplicate of an existing component, please reuse the existing one (`TabComponent`) -- which means add the `disabled` property that you seem to need.

Reply:
Yes, I agree with you I have already tried to use the TabComponent file but it's using class="pane" in HTML and if I try to override the CSS of that class it will override at everyplace. that's the reason I have added this file.

Reply:
@SEAjamieD Could you take a look at this. Should be able to eliminate the duplication. (OK to do this in a follow-up PR.)

Reply:
Yeah we definitely shouldn't be making new components that are this similar.  Worst case lets extend the component we already have.

I've created a new issue to manage this situation here: https://github.com/chef/automate/issues/4866